ruby/app.rb

```diff
@@ -256,23 +318,22 @@
 
 post '/api/create_link_token' do
   begin
-    response = client.link_token.create(
-      user: {
-        # This should correspond to a unique id for the current user.
-        client_user_id: "user-id",
-      },
-      client_name: "Plaid Quickstart",
-      products: ENV['PLAID_PRODUCTS'].split(','),
-      country_codes: ENV['PLAID_COUNTRY_CODES'].split(','),
-      language: "en",
-      redirect_uri: nil_if_empty_envvar('PLAID_REDIRECT_URI'),
+    link_token_create_request = Plaid::LinkTokenCreateRequest.new(
+      {
+        :user => { :client_user_id => 'user-id' },
+        :client_name => 'Plaid Quickstart',
+        :products => ENV['PLAID_PRODUCTS'].split(','),
+        :country_codes => ENV['PLAID_COUNTRY_CODES'].split(','),
+        :language => "en",
+        :redirect_uri => nil_if_empty_envvar('PLAID_REDIRECT_URI')
+      }
```

Comment:
it seems we are mixing syntax here, sometimes we are using `...Request.new({ :foo => bar })` and other times we are using `Request.new` and then setting each attribute one by one. Could we choose just one? I personally prefer this syntax over setting each attribute on a new line. What do you think?